net/shadowsocks-rust/Makefile

```diff
@@ -0,0 +1,54 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
+# Copyright (C) 2017-2020 Yousong Zhou <yszhou4tech@gmail.com>
+# Copyright (C) 2021-2023 ImmortalWrt.org
```

Comment:
Why?

Reply:
Hmmm I'm not sure what do you mean by "why".
This Makefile is modified from yousong's one, so included his copyright.

Reply:
ah okay. forget it :) sorry